samples/sampler/stories/qa/WizardPanels.js

```diff
@@ -360,3 +360,54 @@ _WizardPanelsWithAlert.parameters = {
 		noPanel: true
 	}
 };
+
+
+export const WizardPanelsWithoutSubtitle = (args) => (
+	<WizardPanels
+		current={args['current']}
+		noAnimation={args['noAnimation']}
+		noSteps={args['noSteps']}
+		noSubtitle={args['noSubtitle']}
```

Comment:
Please add this into default sampler (WizardPanels / Panels.header)

Reply:
I don' think we need qa-sampler

Reply:
It's removed and moved to default sampler